src/tests/ftest/datamover/posix_subsets.py

```diff
@@ -54,17 +54,18 @@ def run_dm_posix_subsets(self, tool):
         pool1 = self.create_pool()
 
         # create dfuse containers to test copying to dfuse subdirectories
-        dfuse_cont1 = self.create_cont(pool1)
-        dfuse_cont2 = self.create_cont(pool1)
+        dfuse_cont1 = self.get_container(pool1)
+        dfuse_cont2 = self.get_container(pool1)
         dfuse_cont1_dir = join(self.dfuse.mount_dir.value, pool1.uuid, dfuse_cont1.uuid)
         # destination directory should be created by program
         dfuse_cont2_dir = self.new_posix_test_path(create=False,
             parent=join(self.dfuse.mount_dir.value, pool1.uuid, dfuse_cont2.uuid))
         # Create a special container to hold UNS entries
-        uns_cont = self.create_cont(pool1)
+        uns_cont = self.get_container(pool1)
 
         # Create a testing container
-        container1 = self.create_cont(pool1, True, pool1, uns_cont)
+        container1_path = join(self.dfuse.mount_dir.value, pool1.uuid, uns_cont.uuid, 'uns1')
+        container1 = self.get_container(pool1, path=container1_path)
```

Comment:
(pylint-unexpected-keyword-arg) Unexpected keyword argument 'path' in method call